.github/ISSUE_TEMPLATE/1_bug_report_form.yml

```diff
@@ -88,3 +88,21 @@ body:
         Tip: You can attach images or log files by clicking this area to highlight it and then dragging files in.
     validations:
       required: false
+  - type: textarea
+    attributes:
+      label: Tips for developers
+      description: Please leave this section as-is.
+      value: |
+        Before tackling the bug, please use `git bisect` (see https://git-scm.com/docs/git-bisect)
+        to investigate which PR caused it (you only need to go back as far as
+        commit https://github.com/oppia/oppia/commits/9a334e9).
+        If you find the PR, leave a comment on this issue pointing to it, or
+        just say that it happened before commit 9a334e9 if you could reproduce
+        it there. This will help us fix the issue by reverting the faulty PR.
+
+        Also, if this is your first issue, please make sure to follow
+        https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#choosing-a-good-first-issue
+        and https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#setting-things-up
+        before claiming it. Thanks!
+    validations:
+      required: true
```

Comment:
If this is meant to be a Note and not a textbox, please change it the type to markdown
```yml
type: markdown
```